tests/xtro-sharpie/MacCatalyst-PhotosUI.todo

```diff
@@ -1,10 +0,0 @@
-!missing-protocol! PHContentEditingController not bound
-!missing-type! PHEditingExtensionContext not bound
-## appended from unclassified file
```

Comment:
^ hint ^ everything above this line is part of the backlog and needs a bit more validation